### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by rajmishra-47**
## Description

Describe the purpose and scope of your pull request.

## Related Issues

List any related issues or pull requests that are being addressed.

## Checklist

- [x] I have tested my changes locally.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the project's coding conventions and style guidelines.
- [x] All tests pass successfully.
- [x] I have added/updated relevant unit tests.

## Additional Notes
 I have done a basic contribution i am new to open source please accept my PR
 Include any additional notes or context about your pull request.

### CODE REVIEW

**Comment by rajmishra-47** [comment]
I dont think so that I have created a barch I have just forked this repo and and made changes to that and then send a PR to the main repo I would like to know if its correct.

File: src/App.js [hunk 1]
```diff
@@ -103,6 +103,7 @@ import AnimalsRandomDog from "./api/animals/AnimalsRandomDog";
 import AnimalsShibeOnline from "./api/animals/AnimalsShibeOnline";
 import GeoCodingGetTheData from "./api/geocoding/GeoCodingGetTheData";
 import GeoCodingCountry from "./api/geocoding/GeoCodingCountry";
+import Catfacts from "./api/animals/AnimalCatFacts";
```
Comment (Curtis-Thomas) [inline src/App.js:106]: Update to AnimalsCatFacts

File: src/App.js [hunk 2]
```diff
@@ -159,6 +160,7 @@ const routeConfig = [
   { path: "/AnimalsDash/AnimalsMeowFacts", component: AnimalsMeowFacts },
   { path: "/AnimalsDash/AnimalsRandomDog", component: AnimalsRandomDog },
   { path: "/AnimalsDash/AnimalsShibeOnline", component: AnimalsShibeOnline },
+  { path:"/AnimalsDash/AnimalCatFacts", component:Catfacts},
```
Comment (Curtis-Thomas) [inline src/App.js:163]: Updat to AnimalsCatFacts

File: src/api/animals/AnimalCatFacts.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { Box, useTheme } from "@mui/material";
+import BoxApiInfo from "../../core/main/BoxApiInfo";
+import Endpoints from "../../components/Endpoints";
+import NavBar from "../../navigation/NavBar";
+import BoxTestCrud from "../../tools/components/BoxTestCrud";
+import ApiPageInfo from "../../core/main/ApiPageInfo";
+
+const Catfacts = () => {
```
Comment (Curtis-Thomas) [inline src/api/animals/AnimalCatFacts.js:8]: Updat to AnimalsCatFacts

File: src/api/animals/AnimalCatFacts.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          header="GET"
+          endpoint="fact"
+          description="Retrieve a random cat fact"
+        />
+      </Box>
+    </Box>
+  );
+};
+
+export default Catfacts;
```
Comment (Curtis-Thomas) [inline src/api/animals/AnimalCatFacts.js:45]: Updat to AnimalsCatFacts

File: src/api/animals/Animals1Dash.js [hunk 5]
```diff
@@ -81,6 +81,17 @@ const AnimalDash = () => {
             difficulty="Easy"
           />
         </Grid>
+
+        <Grid item xs={12} sm={6} md={4} lg={3}>
+          <CardRouting
+            cardName="Cat Facts"
+            navigateTo="AnimalCatFacts"
```
Comment (Curtis-Thomas) [inline src/api/animals/Animals1Dash.js:88]: Updat to AnimalsCatFacts

**Review by Curtis-Thomas — CHANGES_REQUESTED** [review]
Hi great work! 

Everything is working, just a few requests on naming conventions I have added as notes.

File: src/App.js (hunk 2 above)
Comment (rajmishra-47) [reply src/App.js]: I have made the necessary changes could you please look into it

File: src/api/animals/AnimalCatFacts.js (hunk 3 above)
Comment (rajmishra-47) [reply src/api/animals/AnimalCatFacts.js]: I have made the necessary changes could you please look into it

File: src/api/animals/AnimalCatFacts.js (hunk 4 above)
Comment (rajmishra-47) [reply src/api/animals/AnimalCatFacts.js]: I have made the necessary changes could you please look into it

File: src/api/animals/Animals1Dash.js (hunk 5 above)
Comment (rajmishra-47) [reply src/api/animals/Animals1Dash.js:88]: I have made the necessary changes could you please look into it

File: src/App.js [hunk 6]
```diff
@@ -103,7 +103,7 @@ import AnimalsRandomDog from "./api/animals/AnimalsRandomDog";
 import AnimalsShibeOnline from "./api/animals/AnimalsShibeOnline";
 import GeoCodingGetTheData from "./api/geocoding/GeoCodingGetTheData";
 import GeoCodingCountry from "./api/geocoding/GeoCodingCountry";
-import Catfacts from "./api/animals/AnimalCatFacts";
+import AnimalCatFacts from "./api/animals/AnimalCatFacts";
```
Comment (Curtis-Thomas) [inline src/App.js:106]: Animals

File: src/App.js [hunk 7]
```diff
@@ -160,7 +160,7 @@ const routeConfig = [
   { path: "/AnimalsDash/AnimalsMeowFacts", component: AnimalsMeowFacts },
   { path: "/AnimalsDash/AnimalsRandomDog", component: AnimalsRandomDog },
   { path: "/AnimalsDash/AnimalsShibeOnline", component: AnimalsShibeOnline },
-  { path:"/AnimalsDash/AnimalCatFacts", component:Catfacts},
+  { path:"/AnimalsDash/AnimalCatFacts", component:AnimalCatFacts},
```
Comment (Curtis-Thomas) [inline src/App.js:163]: Animals

File: src/api/animals/AnimalCatFacts.js [hunk 8]
```diff
@@ -5,7 +5,7 @@ import NavBar from "../../navigation/NavBar";
 import BoxTestCrud from "../../tools/components/BoxTestCrud";
 import ApiPageInfo from "../../core/main/ApiPageInfo";
 
-const Catfacts = () => {
+const AnimalCatFacts = () => {
```
Comment (Curtis-Thomas) [inline src/api/animals/AnimalCatFacts.js:8]: Animals

File: src/api/animals/AnimalCatFacts.js [hunk 9]
```diff
@@ -42,4 +42,4 @@ const Catfacts = () => {
   );
 };
 
-export default Catfacts;
+export default AnimalCatFacts;
```
Comment (Curtis-Thomas) [inline src/api/animals/AnimalCatFacts.js:45]: Animals

File: src/api/animals/Animals1Dash.js [hunk 10]
```diff
@@ -84,7 +84,7 @@ const AnimalDash = () => {
 
         <Grid item xs={12} sm={6} md={4} lg={3}>
           <CardRouting
-            cardName="Cat Facts"
+            cardName="Animal Cat Facts"
             navigateTo="AnimalCatFacts"
```
Comment (Curtis-Thomas) [inline src/api/animals/Animals1Dash.js:88]: Animals

**Review by Curtis-Thomas — CHANGES_REQUESTED** [review]
Should be `Animals` not `animal`

image

File: src/App.js (hunk 6 above)
Comment (rajmishra-47) [reply src/App.js]: Sorry my Bad I have made the necessary changes

File: src/App.js (hunk 7 above)
Comment (rajmishra-47) [reply src/App.js]: Sorry my Bad I have made the necessary changes

File: src/api/animals/AnimalCatFacts.js (hunk 8 above)
Comment (rajmishra-47) [reply src/api/animals/AnimalCatFacts.js]: Sorry my Bad I have made the necessary changes

File: src/api/animals/AnimalCatFacts.js (hunk 9 above)
Comment (rajmishra-47) [reply src/api/animals/AnimalCatFacts.js]: Sorry my Bad I have made the necessary changes

File: src/api/animals/Animals1Dash.js (hunk 10 above)
Comment (rajmishra-47) [reply src/api/animals/Animals1Dash.js]: Sorry my Bad I have made the necessary changes

**Review by Curtis-Thomas — APPROVED** [review]
Great work!

**Review by Curtis-Thomas — APPROVED** [review]
Great Job!

**Pull request merged**